examples/one_degree_simulation.jl

```diff
@@ -170,19 +170,32 @@ ocean.output_writers[:surface] = JLD2OutputWriter(ocean.model, outputs;
                                                   overwrite_existing = true,
                                                   array_type = Array{Float32})
 
+# We also save a series of Checkpointers which will enable the simulation to be restarted 
+# from a later point; ideal for longer runs
+
+directory_checkpointer = "."
+prefix_checkpointer = "one_degree_checkpointer"
+
+simulation.output_writers[:checkpoint] = Checkpointer(ocean.model;
+                                                      schedule = TimeInterval(5days),
+                                                      prefix = prefix_checkpointer,
+                                                      dir = directory_checkpointer,
+                                                      verbose = true,
+                                                      overwrite_existing = true)
+
 # ### Ready to run
 
 # We are ready to press the big red button and run the simulation.
 
 # After we run for a short time (here we set up the simulation with `stop_time = 10days`),
 # we increase the timestep and run for longer.
 
-run!(simulation)
+run!(simulation, pickup=true)
```

Comment:
not sure if we should have the `pickup=true` as the default in the example or explain, e.g., by something along the lines

> If you have run the simulation and you there are checkpointer files produces, next time when you run again you can use `run!(simulation, pickup=true)` to restart from the latest checkpointer.

Reply:
I agree, I don't think we should put `pickup=true`.

The idea is that we only need `pickup=true` on the second time we run the script. So it's not appropriate to put it in the script permanently.

Reply:
But `pickup=true` simply informs you there are no checkpointers and initialises from scratch if no checkpointer files are found. Having `pickup=true` permanently ensures that the checkpointer functionality will always work (including when there are no checkpointers). Otherwise the user would have to manually go in and change from `run!(simulation)` to `run!(simulation, pickup=true)` - something that is inconvenient if you have submitted the job to automatically restart in an HPC scheduler.

Reply:
Agreed.

Reply:
`pickup=true` already deals well with the case where there are no checkpointers, so I don't see why it needs to be removed :D

Reply:
I think it boils down to your different workflows (and as a matter of fact to the workflow the user will be using).

Do you submit job via slurm/qsub?
Do you run scripts on repl?

Reply:
It doesn't make sense. We should just have `pickup=true` as the default if we put it here.

Reply:
> `pickup=true` already deals well with the case where there are no checkpointers, so I don't see why it needs to be removed :D

it makes the assumption that you actually want to pick up, rather than tweak the simulation and re-run which is what you want to do most of the time. This will encourage bad practices / confuse people.